### ISSUE TITLE
VS 2019.16.4.5 and RS 2019.3.2 not working

### ISSUE THREAD

**Issue by Nyrup** [hmemcpy/ReSharper.Xao#21]
This lovely plugin no longer shows the small window from which I can select between view, code-behind and vm. 

Does it hook up to the Resharper.GotoReletaedFiles keybord shortcut or does in have it's own name in VS->Options->Keyboard settings?

**Comment by TeaDrivenDev** [comment]
Yes it does. I also noticed it doesn't currently work in VS, but it works fine in Rider.

**Comment by Nyrup** [comment]
> Yes it does. I also noticed it doesn't currently work in VS, but it works fine in Rider.

I pretty sure it worked in the previous version of Resharper. I was unaware how much I rely on it until it stopped doing it's thing 😊
